src/buildHTML.js

```diff
@@ -174,6 +267,17 @@ export const getTypeOfDomTree = function(node) {
     return null;
 };
 
+export const isTight = function(node) {
+    if (node instanceof domTree.documentFragment) {
+        if (node.children.length) {
+            return isTight(node.children[node.children.length - 1]);
```

Comment:
Why is the recursive call necessary? Is documentFragment just a wrapper? Also, why is the last child a safe assumption?

Reply:
This follows the same pattern as `getTypeOfDomTree`.  Both of these functions need docstrings that explain that if the node is a document fragment then we return the value for the right most node in that fragment.  A document fragment is a tree so it could contain nested nodes hence the recursion.